data-raw/wd_pcon_lad_la_rgn_ctry.R

```diff
@@ -0,0 +1,123 @@
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Created: 05/08/2024, Cam Race
+#
+# This script creates the wd_pcon_lad_la_reg_ctry data set
+#
+# If updating the data, follow these steps:
+# 1. Load the package using `devtools::load_all(".")`
+# 2. Add in an extra year for the new year into this script within the
```

Comment:
I've figured out what you mean, but it took me a bit of re-reading. It'd be simpler to describe if there was a specific variable along the lines of:
`years_wd_pcon_lad_la <- c(17, 19:24)`

Reply:
Done (and updated instructions a bit)